apps/chat-e2e/src/tests/chatSelectionFunctionality.test.ts

```diff
@@ -0,0 +1,251 @@
+
+    await dialTest.step('Create chat1 and chat2', async () => {
+      firstConversation = conversationData.prepareDefaultConversation();
+      conversationData.resetData();
+      secondConversation = conversationData.prepareDefaultConversation();
+      await dataInjector.createConversations([
+        firstConversation,
+        secondConversation,
+      ]);
+      await localStorageManager.setSelectedConversation(secondConversation);
+    });
+
+    await dialTest.step(
+      'Click on chat2 -> its history is opened on the central part of the screen',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded();
+      },
+    );
+
+    await dialTest.step('Hover over chat1', async () => {
+      await conversations.getEntityByName(firstConversation.name).hover();
+      await conversationAssertion.assertEntityBackgroundColor(
+        { name: firstConversation.name },
+        Colors.backgroundAccentSecondary,
+      );
+      await conversationAssertion.assertEntityDotsMenuState(
+        { name: firstConversation.name },
+        'visible',
+      );
+    });
+
+    await dialTest.step(
+      'Click on the menu and hover over any item at the bottom (e.g. Delete)',
+      async () => {
+        await conversations.openEntityDropdownMenu(firstConversation.name);
+        await conversations
+          .getDropdownMenu()
+          .menuOption(MenuOptions.delete)
+          .hover();
+        await conversationAssertion.assertEntityBackgroundColor(
+          { name: firstConversation.name },
+          Colors.backgroundAccentSecondary,
+        );
+        await conversationAssertion.assertEntityBackgroundColor(
+          { name: secondConversation.name },
+          Colors.backgroundAccentSecondary,
+        );
+        await conversationAssertion.assertEntityDotsMenuState(
+          { name: secondConversation.name },
+          'hidden',
+        );
+      },
+    );
+
+    await dialTest.step('Click on Rename, rename and confirm', async () => {
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.rename);
+      firstConversation.name = 'Renamed chat';
+      await conversations.editConversationNameWithTick(firstConversation.name);
+      await conversations.getEntityByName(firstConversation.name).waitFor();
+      await conversationAssertion.assertSelectedConversation(
+        secondConversation.name,
+      );
+      await conversationAssertion.assertEntityState(
+        { name: firstConversation.name },
+        'visible',
+      );
+    });
+
+    await dialTest.step('Click on Compare', async () => {
+      await conversations.openEntityDropdownMenu(firstConversation.name);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.compare);
+      await conversationAssertion.assertSelectedConversation(
+        firstConversation.name,
+      );
+      await expect
```

Comment:
please, use ` await conversationAssertion.assertEntityState()` instead

Reply:
It is not neccesary